_includes/svg/ai2-logo.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 35 27">
+  <path d="M15.8,4.7l1.1,0.5l-0.9,0.5c-0.1,0-0.1,0.1-0.1,0.2c0,0.1,0.1,0.1,0.1,0.1l1.2,0.2L16.3,7 c-0.1,0.1-0.1,0.1,0,0.2s0.1,0.1,0.2,0.1l1.3-0.3l-0.4,0.8c0,0.1,0,0.1,0.1,0.2c0.1,0.1,0.1,0.1,0.2,0l0.6-0.3 c-0.2-0.5-0.3-1-0.5-1.5c-0.1-0.3-0.3-0.5-0.3-0.8c-0.3-0.6-0.2-1.2,0.1-1.7c0.4-0.6,0.8-0.9,1.5-1.1c0.7-0.2,1.4,0,2,0.5 c0.4,0.4,0.6,0.8,0.7,1.3s0,1.1-0.3,1.5C21.2,6.2,21,6.6,20.9,7c-0.1,0.2-0.1,0.3-0.2,0.5l0.7,0.4h0.1c0.1,0,0.1,0,0.1-0.1 c0.1-0.1,0.1-0.1,0.1-0.2l-0.4-0.8l1.3,0.3c0.1,0,0.1,0,0.2-0.1c0.1-0.1,0-0.1,0-0.2L21.9,6l1.2-0.2c0.1,0,0.1-0.1,0.1-0.1 c0-0.1,0-0.1-0.1-0.2L22.2,5l1.1-0.5c0.1,0,0.1-0.1,0.1-0.1c0-0.1-0.1-0.1-0.1-0.1L22,4.1l1.1-0.9c0.1-0.1,0.1-0.1,0.1-0.2 S23,3,23,3l-1.3,0.3L22.5,2c0.1-0.1,0.1-0.1,0-0.2c-0.1-0.1-0.1-0.1-0.2-0.1l-1.4,0.8L21.3,1c0-0.1,0-0.1-0.1-0.2 c-0.1,0-0.1,0-0.2,0.1L20,2.2l-0.4-1.8c0-0.1-0.1-0.1-0.1-0.1s-0.1,0.1-0.1,0.1L19,2.3L18,0.9c-0.1-0.1-0.2-0.1-0.2-0.1 S17.7,1,17.7,1L18,2.7l-1.4-0.8c-0.1-0.1-0.1,0-0.2,0.1c-0.1,0.1-0.1,0.1,0,0.2l0.8,1.2L16,3c-0.1,0-0.1,0-0.2,0.1 c0,0.1,0,0.1,0.1,0.2l1.1,0.9l-1.2,0.2c-0.1,0-0.1,0.1-0.1,0.1C15.7,4.6,15.7,4.6,15.8,4.7z" />
+  <path d="M27.8,23.5l3.9-5.9c1.4-2.2,2.1-3.9,2.1-5.4c0-1.6-0.5-2.8-1.6-3.7c-1-0.8-2.7-1.3-4.5-1.3l0,0 c-0.1,0-0.1,0-0.1,0c-0.1,0-0.1,0-0.1,0l0,0c-5,0.1-10.3,3.2-14.9,6.8l-2.4-6.6H6.8L0.1,26.7h3.7c0,0,2.7-3.6,6.7-7.5l2.5,7.5h3.7 l-3.5-10c1.4-1.2,2.9-2.3,4.5-3.3v13.3h3.4V11.6c2.2-0.9,4.4-1.5,6.5-1.5c1.7,0,2.6,0.7,2.6,2.2c0,1.2-0.6,2.7-1.8,4.5l-6.1,8.8v1 h12.4v-3H27.8z M6.1,19.9l2.4-7.2l1.2,3.7C8.4,17.6,7.2,18.8,6.1,19.9z" />
+  <path d="M18.3,8.5c0.1,0.2,0.1,0.3,0.1,0.5c0,0.1,0,0.1,0.1,0.1c0.7-0.4,1.5-0.8,2.2-1.2c0-0.1,0-0.1,0.1-0.1 c0-0.1,0-0.1,0.1-0.1h-2.6C18.2,8,18.3,8.2,18.3,8.5z" />
+</svg>
```

Comment:
Can't this just be a static PNG, which should compress better / be smaller? Feel free to compare sizes, if the SVG is smaller I'm ok w/ that.

Reply:
Not a huge fan of PNGs for icons because they don't scale well and managing retina versions is kind of a pain. Also I prefer to use inline SVG over `.svg` assets so I can transition fills and stuff.

And just in case it wasn't obvious, I'm actually including this SVG code in the markup via Jekyll includes. I'm not pointing to these files as standalone image assets.

That said, if you want me to rebuild these as part of a PNG sprite I will. Will take a little time though. I just love the clean, scalable vector-ness and fine-tune control of inline SVG. :) I figured for a small page like this, we're not losing too much by doing it this way. But it's your call.